### PR TITLE
Win Condition

### PR DESCRIPTION
**Pull request by lsizemore8**
Win Condition correctly displays the word "victory" above the goal and resets after a brief period of time. 

Note: because of the different coordinate systems for gameCanvas, victory has to be drawn in what the game sees as the upper left corner, despite the camera centering on that position.

### CODE REVIEW

File: core/src/com/fallenflame/game/GameEngine.java [hunk 1]
```diff
@@ -253,6 +264,11 @@ public boolean preUpdate(float dt) {
             listener.exitScreen(this, EXIT_QUIT);
             return false;
         }
+        else if (countdown > 0) {
+            countdown--;
+        } else if (countdown == 0) {
+            reset();
+        }
```
Comment (BryanGraeser) [inline core/src/com/fallenflame/game/GameEngine.java:273]: Why are we resetting if countdown is reached?

**Review by BryanGraeser — COMMENTED** [review]
Why are you changing InputController in this PR?

I shouldn't be, that's strange. Likely a merge conflict gone wrong

File: core/src/com/fallenflame/game/GameEngine.java (hunk 1 above)
Comment (lsizemore8) [reply core/src/com/fallenflame/game/GameEngine.java:273]: Countdown is -1 until we win or lose. It's used purely to show "Victory" or "You Died" for a certain amount of time (ie until countdown reaches 0). Once it reaches 0, reset

**Comment by BryanGraeser** [comment]
gtg

**Review by BryanGraeser — APPROVED** [review]
gtg

**Pull request merged**